docs_source_files/content/driver_idiosyncrasies/shared_capabilities.de.md

```diff
@@ -49,4 +49,27 @@ as described in the table below:
 | -------- | ----------- | ----- |
 | normal | complete | Used by default, waits for all resources to download |
 | eager | interactive | DOM access is ready, but other resources like images may still be loading |
-| none | Any | Does not block WebDriver at all |
\ No newline at end of file
+| none | Any | Does not block WebDriver at all |
+
+## platformName
+
+This identifies the operating system at the remote-end, 
+fetching the `platformName` returns the OS name. 
+
+In could-based providers, 
```

Comment:
cloud

Reply:
Thanks a lot @cgoldberg. I have updated in all translated pages!